cli/src/main/java/com/liferay/blade/cli/command/InitCommand.java

```diff
@@ -95,6 +95,10 @@ public void execute() throws Exception {
 			profileName = initArgs.getBuild();
 		}
 
+		if (!"maven".equals(profileName) && !"gradle".equals(profileName)) {
```

Comment:
Hey Vernon,

This needs to be more sophisticated check like the following:

1. check to see if there are registered commands that have this profile name
2. check to see if there is a custom template that ends with `*project.templates.workspace.<profilename>`

If neither one of these is true, then we could print out a warning.  Because either of the cases above and this is a valid profile name.

Reply:
Let me know or Chris if you need some more guidance of how to possibly implement this.

Reply:
No problem, thanks for explaining.  I knew it needed more ... just did not know where to start.  Thanks for the tips.

Reply:
Sure thing! Thanks for your patience for me to get around to explaining it.